schemas/server-common.json

```diff
@@ -56,14 +56,12 @@
     "path": {
       "$anchor": "zowePath",
       "type": "string",
-      "pattern": "^([^\0]){1,1024}$",
```

Comment:
The regex was deleted:
- You can not use null terminator in the middle of the string, first null term will terminate the string as expected
- So the deleted regex (which is anyway wrong due to the missing backslash) is kind of useless

The `zwe config get` for this:

```
  zowe:
    setup:
      dataset:
        prefix: MY.ZOWE
        parmlib: ${{ zowe.setup.dataset.prefix }}.PARMLIB
        proclib: ${{ zowe.setup.dataset.prefix }}.PROCLIB
        jcllib: ${{ zowe.setup.dataset.prefix }}.JCLLIB
        authLoadlib:  ${{ zowe.setup.dataset.prefix }}.TEST1
        authPluginLib:  ${{ zowe.setup.dataset.prefix }}.TEST2
      security:
        stcs:
          zowe: MYZOWE
          zis:  ${{ zowe.setup.security.stcs.zowe.substr(0,2) }}ZIS
          aux:  ${{ zowe.setup.security.stcs.zowe.substr(0,2) }}AUX
    useConfigmgr: true
    runtimeDirectory: "/path/to/myZowe\x00 The rest of the string ignored..."
    
```

Will return:

```
{
   "setup":{
      "dataset":{
         "prefix":"MY.ZOWE",
         "parmlib":"MY.ZOWE.PARMLIB",
         "proclib":"MY.ZOWE.PROCLIB",
         "jcllib":"MY.ZOWE.JCLLIB",
         "authLoadlib":"MY.ZOWE.TEST1",
         "authPluginLib":"MY.ZOWE.TEST2"
      },
      "security":{
         "stcs":{
            "zowe":"MYZOWE",
            "zis":"MYZIS",
            "aux":"MYAUX"
         }
      }
   },
   "useConfigmgr":true,
   "runtimeDirectory":"/path/to/myZowe"
}
```